app/helpers/merge_requests_helper.rb

```diff
@@ -22,14 +22,18 @@ def merge_request_pending_since(mr)
     "pending for #{time}"
   end
 
-  def patch_ci_icon(patch, only_cached = nil)
+  def patch_ci_status(patch, only_cached = nil)
     if patch.pass?
-      content_tag(:i, '', class: 'tipped fa fa-check ok', 'data-tip' => 'CI passed')
+      'review-list--success'
     elsif patch.canceled?
-      content_tag(:i, '', class: 'tipped fa fa-ban', 'data-tip' => 'CI canceled')
-    elsif only_cached && patch.failed?
-      content_tag(:i, '', class: 'tipped fa fa-remove fail', 'data-tip' => 'CI failed')
-    elsif only_cached
+      'review-list--canceled'
+    else only_cached && patch.failed?
```

Comment:
Ooops, you meant `elsif` instead of `else`, I noticed just after merge. anyway, typo already fixed :-)